app/(company)/about/components/BuildWithUsSection.js

```diff
@@ -0,0 +1,37 @@
+import { BuildWithUsGridItems } from './BuildWithUsGridItems';
+
+const BuildWithUsHeaderAndBlurb = () => <div className="xl:col-span-2">
+    <h2 className="text-4xl mb-4 md:text-5xl leading-tight font-medium md:text-6xl leading-snug">Build
+    a better <br className="hidden md:inline" />democracy <br
+      className="md:hidden" /><span
+      className="text-tertiary-main">with us</span></h2>
+  <p className="text-xl font-medium leading-7 mb-8 md:max-w-[466px] md:text-2xl leading-8">Ready
+    to join the movement? Support candidates, run for office or join our Discord
+    community of like-minded individuals.</p>
+  </div>
```

Comment:
are you using the same code linter as we do? is you IDE respects prettier?

Reply:
`npm run lint` exits successfully? 🤷 
<img width="249" alt="image" src="https://github.com/thegoodparty/gp-webapp/assets/496983/1b95d2a9-ce97-4d7f-9d9b-195728210cd9">

Reply:
Same result w/ `next lint --fix`? 🤷 
<img width="570" alt="image" src="https://github.com/thegoodparty/gp-webapp/assets/496983/8986f0c0-6d42-49b5-bb83-727e81c723d3">